e2e/_suites/fleet/fleet.go

```diff
@@ -391,17 +408,20 @@ func (fts *FleetTestSuite) processStateChangedOnTheHost(process string, state st
 
 	serviceName := agentInstaller.Service // name of the service
 
+	profileService := deploy.NewServiceRequest(profile)
+	imageService := deploy.NewServiceRequest(common.ElasticAgentServiceName).WithFlavour(agentInstaller.Image)
+
 	if state == "started" {
-		return installer.SystemctlRun(profile, agentInstaller.Image, serviceName, "start")
```

Comment:
In my branch these are actually abstracted out to the installer service as `Start` and `Stop` and is specific to the service we are installing (this could differ with other services that may not use systemctl)